src/Form/Form.js

```diff
@@ -227,6 +230,14 @@ class Form extends Util.mixin(BindMixin) {
     ]);
 
     return definition.map((formControlOption, i) => {
+      if (React.isValidElement(formControlOption)) {
```

Comment:
Also a comment here as to why we're wrapping the `formControlOption` is it so it has a `key`?